manager/allocator/cnmallocator/drivers_network_windows.go

```diff
@@ -3,12 +3,17 @@ package cnmallocator
 import (
 	"github.com/docker/libnetwork/drivers/overlay/ovmanager"
 	"github.com/docker/libnetwork/drivers/remote"
+	"github.com/docker/libnetwork/drivers/windows"
 	"github.com/docker/swarmkit/manager/allocator/networkallocator"
 )
 
 var initializers = []initializer{
 	{remote.Init, "remote"},
 	{ovmanager.Init, "overlay"},
+	{windows.GetInit("transparent"), "transparent"},
+	{windows.GetInit("l2bridge"), "l2bridge"},
+	{windows.GetInit("l2tunnel"), "l2tunnel"},
+	{windows.GetInit("nat"), "nat"},
```

Comment:
Is there anything else I need to do to get this merged?
Thanks again for your time.